massgov/mayflower
# React/dp 9851  blur input

## Description
Blurs the search text input when enter is pressed.

## Related Issue / Ticket
https://jira.mass.gov/browse/DP-9851

## Steps to Test

1. check this branch out & run `npm run build`
2. in massgov-search `npm install && npm link <path to your mayflower/react> && npm start`
3. load search `http://localhost:3000/`
4. type a search keyword & press enter -- the text input should no longer keep focus

## Screenshots
![blur-on-enter](https://user-images.githubusercontent.com/38226003/44594288-b85e0280-a793-11e8-966c-2df5ac111725.gif)

## Additional Notes:
* This cannot be done on the mf/react TextInput element because causing a blur on its key press results in subsequent events not happening (form submit button click, for example)
** see commit log in this PR for attempt in that direction or this fiddle (shows form submit not happening on Enter key press): http://jsfiddle.net/pmallett/rf4ko5bj/
* also, mf/react TextInput is not used in any other component than the unused "SearchBannerForm" at this time

#### Impacted Areas in Application

*  Search input in HeaderSearch

#### @TODO


#### Today I learned...
